### ISSUE TITLE
stacking bug when animated with children of fixed elements

### ISSUE THREAD

**Issue by zevdg** [kamranahmedse/driver.js#133]
the stage gets rendered over top of the element resulting in an empty white box.

codepen: https://codepen.io/zevdg/pen/QZazyO

possibly related to #126 and/or #97

**Comment by rrfranco92** [comment]
Do you managed to get a fix on this? Same thing happening to me with a fixed position element.

**Comment by Eggwise** [comment]
Also having this issue... any fix ?

**Comment by ChucKN0risK** [comment]
I have the same issue when a parent element has `position: sticky`. Does anyone have a solution for this?

**Comment by ChucKN0risK** [comment]
I've noticed that `driver.js` detected my fixed/sticky HTML element and added on it the `.driver-fix-stacking` CSS class.

I managed to solve the issue by adding `position: relative` to this class like so:
```css
.driver-fix-stacking {
  position: relative !important;
}
```

Hope it helps 😉

**Comment by dingyaguang117** [comment]
+1

I eventually change fixed to absolute.

**Comment by kapooramit** [comment]
I have the same issue but I do not see the ```.driver-fix-stacking``` class added to my elements. What else could cause this? I would really want to enable animation.

**Comment by zengkan0703** [comment]
same one.
Change ```fixed``` to ```absolute || relative``` will affect the layout, so I have not find way to solve this problem

**Comment by dipiana** [comment]
I encounter the same problem.

@kamranahmedse always references to #112 with the solution being #127.
I don't see how #127 solves the issue? I could not make it work with a nav which is fixed.

I took one example (from #95, where #127 is referenced as a solution) and put it into CodePen with the latest version of driver.js - the issue still occurs. How could #127 fix that?

Would be great to find a working solution. Thanks a lot in advance.

**Comment by leducson** [comment]
@zevdg For animate = true, you can overwrite the following css class which will solve your problem

```
div#driver-highlighted-element-stage, div#driver-page-overlay {
  background: transparent !important;
  outline: 5000px solid rgba(0, 0, 0, .75)
}
```
You can try it directly with the example on your codepen
https://codepen.io/zevdg/pen/QZazyO

**Comment by tw2as** [comment]
@kamranahmedse - would you consider implementing this hack (https://github.com/kamranahmedse/driver.js/issues/133#issuecomment-549714982) into driver.js?

**Comment by Tarilonte** [comment]
I managed to workaround it by temporarily changing the position of the element while higlighting it.

**Comment by guguji5** [comment]
It doesn't work in antd Drawer, add `  style={{ position: 'absolute' }} ` on Drawer can fix it.

**Comment by brizer** [comment]
> I ended up using it without animation if I recall correctly.

me too. `animate: false` is works.

**Comment by kamranahmedse** [comment]
We just released [v1.0](https://github.com/kamranahmedse/driver.js/releases/tag/1.0.3) of driver.js which has been rewritten from the ground up. This issue should no longer exist. Please have a [look at the docs](https://driverjs.com/) and do let me know if you face any issues.
